3-reveal/migrations/3-views/deploy/plans_with_case_classification_view.psql

```diff
@@ -0,0 +1,73 @@
+-- Deploy reveal_database_views:plans_with_case_classification_view to pg
+-- requires: reveal_transaction_tables:events
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS pending_:schema .plans_materialzied_view AS
+SELECT
+	DISTINCT ON (plans.identifier, plan_jurisdiction.jurisdiction_id)
+	uuid_generate_v5(
+	'6ba7b810-9dad-11d1-80b4-00c04fd430c8',
+	concat(plans.identifier, plan_jurisdiction.jurisdiction_id)) AS id,
+	plans.identifier AS plan_id,
+	plans.title AS plan_title,
+	plans.name AS plan_name,
+	plans.status AS plan_status,
+	tree.jurisdiction_id AS jurisdiction_id,
+	plans.fi_status AS plan_fi_status,
+	plans.fi_reason AS plan_fi_reason,
+	plans.date AS plan_date,
+	plans.effective_period_start AS plan_effective_period_start,
+	plans.effective_period_end AS plan_effective_period_end,
+	plans.intervention_type AS plan_intervention_type,
+	plans.version AS plan_version,
+	tree.jurisdiction_parent_id AS jurisdiction_parent_id,
+	tree.jurisdiction_name AS jurisdiction_name,
+	tree.jurisdiction_geometry AS jurisdiction_geometry,
+	tree.jurisdiction_depth AS jurisdiction_depth,
+	tree.jurisdiction_path AS jurisdiction_path,
+	tree.jurisdiction_name_path AS jurisdiction_name_path,
+	tree.jurisdiction_root_parent_id AS jurisdiction_root_parent_id,
+	tree.jurisdiction_root_parent_name AS jurisdiction_root_parent_name,
+  events.details->>'case_classification' AS case_classification
```

Comment:
Could we format this line like the others?

Reply:
Looks fine on my editor.

Reply:
![Oo](https://user-images.githubusercontent.com/372073/109318449-dd4cf800-785e-11eb-82d4-bbbf18c77535.png)


I think that there might be an issue with your editor.